test/query.coffee

```diff
@@ -162,22 +162,22 @@ _([
 
     it 'exists true with where works', (done) ->
       @clever.School.find().where('name').exists(true).count().exec (err, count) ->
-        assert.equal count, 4
+        assert.equal count, 3
```

Comment:
Discussed offline; these expectations are changed here because the demo district changed. We should rewrite these tests to not hit the production API in another PR.